frontend/src/views/Org/UserPage/components/UserProjectsSection/UserGroupsRow.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/jsx-no-useless-fragment */
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { IconButton, Td, Tooltip, Tr } from "@app/components/v2";
+import { TGroupWithProjectMemberships } from "@app/hooks/api/groups/types";
+import { UsePopUpState } from "@app/hooks/usePopUp";
+
+type Props = {
+  group: TGroupWithProjectMemberships;
+  handlePopUpOpen: (popUpName: keyof UsePopUpState<["removeUserFromGroup"]>, data?: {}) => void;
+};
+
+export const UserGroupsRow = ({ group, handlePopUpOpen }: Props) => {
+  return (
+    <>
+      <Tr
+        className="group h-10 cursor-pointer transition-colors duration-100 hover:bg-mineshaft-700"
+        key={`user-project-membership-${group.id}`}
+      >
+        <Td>{group.name}</Td>
+        <Td>
+          {true && (
```

Comment:
Why do we need `true && (`? Can't we just omit this?

Reply:
Oh wow good catch! That was a left-over from testing